### PR TITLE
FIX: Environment error on CircleCI

### PR DESCRIPTION
**Pull request by esavary**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -14,19 +14,17 @@ jobs:
...
           name: Activate virtual environment
-          command: myenv/bin/activate
+          command: source myenv/bin/activate
       - run:
-          name: Upgrade pip and install requirements
-          command: |
-            myenv/bin/pip install --upgrade pip
-            myenv/bin/pip install -r requirements.txt
+          name: Install requirements
+          command: pip install -r requirements.txt
```
Comment (oesteban) [inline .circleci/config.yml]:
```suggestion
          command: python -m pip install -r requirements.txt
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -14,19 +14,17 @@ jobs:
...
       - run:
           name: Build preview
-          command: myenv/bin/mkdocs build --site-dir _SOPs_preview
+          command: mkdocs build --site-dir _SOPs_preview
```
Comment (oesteban) [inline .circleci/config.yml]:
```suggestion
          command: python -m mkdocs build --site-dir _SOPs_preview
```

**Review by oesteban — APPROVED** [review]
Can you give my suggestions a try? Using `python -m` may make using the venv's python more robust across versions

**Comment by esavary** [comment]
I tested your suggestion, and it's working. Can I proceed with the merge?

**Comment by oesteban** [comment]
> I tested your suggestion, and it's working. Can I proceed with the merge?

Thanks!

Unfortunately no - if you go to the generated docs (https://output.circle-artifacts.com/output/job/b3f24179-32b3-405e-8d14-cb207d8b881c/artifacts/0/_SOPs_preview/index.html), for some reason the other sections are not uploaded as artifacts and the navigation doesn't work.

**Comment by oesteban** [comment]
I'm wrong - it's an mkdocs problem. The files are there.

**Pull request merged**